src/Ryujinx.Ava/UI/Windows/DownloadableContentManagerWindow.axaml.cs

```diff
@@ -61,7 +61,7 @@ private void Close(object sender, RoutedEventArgs e)
             ((ContentDialog)Parent).Hide();
         }
 
-        private void RemoveDLC(object sender, RoutedEventArgs e)
+        private void RemoveDlc(object sender, RoutedEventArgs e)
```

Comment:
```suggestion
        private void RemoveDLC(object sender, RoutedEventArgs e)
```

or `RemoveDownloadableContent`